compilers/esm.js

```diff
@@ -101,7 +101,7 @@ exports.compile = function(load, opts, loader) {
       lowResolutionSourceMap: opts.lowResSourceMaps
     });
 
-    var tree = load.metadata.parseTree || compiler.parse(load.source, load.path);
+    var tree = load.metadata.parseTree || compiler.parse(source, load.path);
```

Comment:
Actually, changing the one on line 149 fixed this for me locally, whereas changing this one did not.